constant.py

```diff
@@ -0,0 +1,47 @@
+import os
+IPFS_URL = {
+    'url': 'ipfs.singularitynet.io',
+    'port': '80',
+    'protocol': 'http'
+}
+NETWORKS_NAME = {
```

Comment:
Dont need this as a config entry, We can build this map by iterating thru NETWORKS